src/assets/scss/components/_components.scss

```diff
@@ -7,5 +7,6 @@
 @import "./accountBenefits";
 @import "./failedVerificationModal";
 @import "./oktaSigninWidget";
+@import "./loading";
```

Comment:
This change and the rename of the loading stylesheet won't be necessary after #1196 merges.